.github/workflows/test.yml

```diff
@@ -38,15 +40,34 @@ jobs:
           cache: npm
       - run: npm ci
       - run: npm test
-    
-  test:
+
+  test-deno:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: denoland/setup-deno@v2
+        with:
+          deno-version: v2.x # Run with latest stable Deno.
+      - run: deno install
+      - run: deno test --allow-env --allow-read --allow-net
+
+  test-bun:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: oven-sh/setup-bun@v2
+      - run: bun install
+      - run: bun test
+            
+  validate-typescript:
```

Comment:
Because this job was renamed, it will require the intervention of an admin to merge this

Reply:
is it better now?

Reply:
yeah, that works